tuning/data/setup_dataprocessor.py

```diff
@@ -491,6 +495,72 @@ def dump_dataset(
         raise RuntimeError(f"Failed to dump dataset due to error {e}") from e
 
 
+def process_dataargs_odm(
+    data_args: DataArguments,
+    tokenizer: AutoTokenizer,
+    train_args: TrainingArguments,
+    is_padding_free: bool = False,
+    processor: AutoProcessor = None,
+    odm_config: ODMConfig = None,
+    train_dataset: Dict = None,
+    eval_dataset: Dict = None,
```

Comment:
can we rename this to `reward_dataset` and pass `eval_dataset` here after mentioning a comment in the code